tools/ios.toolchain.cmake

```diff
@@ -335,6 +335,9 @@ if(USED_CMAKE_GENERATOR MATCHES "Xcode")
   endif()
 endif()
 
+# Required from Xcode 12 on, because it now builds for arm macbooks and we don't support that.
+set(CMAKE_XCODE_ATTRIBUTE_EXCLUDED_ARCHS[sdk=iphonesimulator*] "arm64")
```

Comment:
I was looking for that cmake argument but did not find it, nice!

Reply:
Yeah you were right on that from the beginning :100: